devices.js

```diff
@@ -1300,6 +1300,14 @@ const devices = [
         description: 'Light controller',
         extend: preset.light_onoff_brightness_colortemp(),
     },
+    {
+        fingerprint: [{modelID: 'TS0504A', manufacturerName: '_TZ3000_nzbm4ad4'}],
+        zigbeeModel: ['TS0504A'],
```

Comment:
Please remove the zigbeeModel line and check if it still works.

Reply:
Yep, still working.